### DISCUSSION TITLE
changing ploidy values do not change call outcomes

### DISCUSSION THREAD

**Discussion by CKZan** [etal/cnvkit#1138] [General]
Running CNVkit call with default pramaters I got excessive loss call so I have tried using –ploidy 3 then –ploidy 4 and 5. I was expecting some changes on the results but they were the same. Would you help to explain what I have done wrong? The run command is 
cnvkit/0.9.1/cnvkit.py call xx.cns –ploidy 3 -o xx.call.cns

**Comment by tetedange13** [comment]
Hi @CKZan ,

Are you really running CNVkit v0.9.1? 
=> It is currently at v0.9.9, so please update and see if you still got this issue ?

Hope this helps !
Felix.

**Comment by CKZan** [comment]
Hi Felix,
Yes I used cnvkit/0.9.9/cnvkit.py call and got the same results. 
For example by using default I got gain covered 100Mb and loss covered 150Mb then with ploidy 3,4 or 5 I got the same gain = 100Mb and loss = 150Mb where individual segment cn call did change like below

 	                chromosome	start	          end	       cn
 Default	        10	120052576	120140562	4
 Ploidy =3	10	120052576	120140562	6
 Default	        11	60963220	60970805	5
 Ploidy =3	11	60963220	60970805	8

These increace “cn” are only observed in gain regions.

From reading CNVkit document, log2 is scaled by given ploidy if ploidy is higher then shouldn’t I get lower copy number call?  
I also used --center median but got more loss then gain. 
For the purpose of reducing loss call region without twisting -m threshold -t log2 value is there any parameter option that helps to reduce loss call?

Thank you for your help!

**Comment by tetedange13** [comment]
Hi @CKZan,
<br>

Thanks for updating !

If I understand correctly :
1. Your initial issue was that you got "excessive loss call"
=> What do you mean by that ? Could you share a `scatterplot` of your data ?
=> What type of input data do you have ? WGS ? Targetted ? If so, which enrichment method, hybrid-capture or amplicon ?

2. Trying to tweak `--ploidy` param to reduce your number of loss calls, you noticed that higher ploidy value actually results in higher CN values in gain regions

Could you also share corresponding log2ratio value for both gained segments you showed ? ("chr10:120052576-120140562" and "chr11:60963220-60970805")
<br>

Thanks again !
Have a nice day,
Felix.
